src/3k-mapper.xml

```diff
@@ -438,8 +443,12 @@ end</script>
                 <Alias isActive="yes" isFolder="no">
                     <name>Show Exits</name>
                     <script>--show room exits
-tmap:echo(getRoomExits(tmap.lastId))
-tmap:echo(&quot;special exits:\n&quot;)
+local exits = getRoomExits(tmap.lastId)
+echo(&quot;Normal Exits:\n&quot;)
+for k,v in pairs(exits) do
+  echo(&quot;   &quot; .. k .. &quot; -&gt; &quot; .. v .. &quot;\n&quot;)
+end
+tmap:echo(&quot;Special Exits:\n&quot;)
 tmap:echo(getSpecialExitsSwap(tmap.lastId))</script>
```

Comment:
Oy... all issues to file at https://github.com/Chris7/3k-Scripts if you'd like. I didn't write this script, just updating it to latest, we're already shipping it with Mudlet!